alfasim_sdk_api/detail/bootstrap_linux.h

```diff
@@ -10,74 +10,70 @@
 
 #define MAX_PATH 4096
 
-struct alfasim_sdk_bootstrap {
-    alfasim_sdk_bootstrap()
-    {
-        char SO_FILENAME[] = "/alfasim_plugins_api.so";
+inline void alfasim_sdk_open(ALFAsimSDK_API* api)
+{
+    char SO_FILENAME[] = "/alfasim_plugins_api.so";
```

Comment:
Suggestion to either fail or to ignore (possibly with a warning) if it was already "opened" and `alfasim_sdk_open` was called again.